tests/TestCase/Error/ExceptionRendererTest.php

```diff
@@ -761,6 +761,9 @@ public function testMissingLayoutPathRenderSafe()
         $ExceptionRenderer->render();
         $this->assertEquals('', $ExceptionRenderer->controller->viewBuilder()->layoutPath());
         $this->assertEquals('Error', $ExceptionRenderer->controller->viewBuilder()->templatePath());
+
+        // Just to ensure the callback has actually been triggered, so we're actually testing something:
+        $this->assertTrue($eventTriggered);
```

Comment:
This is probably redundant. I added this check that is some sort of a home-made expectation to prevent the problem of the callback not being triggered from happening again.

I don't know if this can be done in some other, more elegant way… 😕